src/test/java/uk/ac/vfb/geppetto/test/CrossDataSourceVFBQueryTest.java

```diff
@@ -185,17 +195,23 @@ public void testRunQuery() throws GeppettoInitializationException, GeppettoVisit
 
 		Variable variable = geppettoModelAccess.getPointer("FBbt_00003748").getElements().get(0).getVariable();
 
-		int count = aberDataSource.getNumberOfResults(getRunnableQueries(model.getQueries().get(avQ.get("partsof")), variable));
-		Assert.assertEquals(84, count);
+		int count = owleryDataSource.getNumberOfResults(getRunnableQueries(model.getQueries().get(avQ.get("partsof")), variable));
+
+		try{
+			Assert.assertTrue(84<count);
+		}catch (AssertionError e) {
+			System.out.println("Fail: only " + count + " results returned, there should be more than than 84 results");
```

Comment:
@ddelpiano what's the advantage of catching and throwing? Custom message?

Reply:
@tarelli yep, mainly to get an error message for the test, this part has been changed by @Robbie1977

Reply:
@tarelli does your version of the unit test have a greater than test? I think they added something but last time i checked we didn't have.